lib/cli/index.js

```diff
@@ -77,6 +77,17 @@ module.exports = function(options) {
   const Project = require('../models/project');
   let config = getConfig(options.Yam);
 
+  let envVariables = config.get('environment') || {};
+  for (let key in envVariables) {
+    let value = envVariables[key];
+    // * env variables passed directly should have more priority over .ember-cli.
+    // * process.env will have only string values. Therefore, check if the value
+    //   in .ember-cli is truthy and then assign to process.env.
+    if (!process.env.hasOwnProperty(key) && value) {
```

Comment:
I think I would do `if (!(key in process.env)) {}`, also why does `value` need to be truthy? I'm concerned about `null` and `0` as values, where a normal `process.env.something = null` would coerce to `'null'`...

Reply:
That makes sense, thanks 👍